ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/services/FragmentCreator.java

```diff
@@ -2,7 +2,9 @@
 
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesFragment;
 
+import java.util.Map;
+
 public interface FragmentCreator {
 
-    LdesFragment createFragment();
+    LdesFragment createFragment(Map<String, String> ldesFragmentConfig);
```

Comment:
Pass FragmentConfig, not the map

Reply:
The issue was with dependencies.. but will be fixed in fragmentation story